installation/install_tools.sh

```diff
@@ -13,6 +13,10 @@ tools/install_package bzr
 # for the virtual machine manager
 tools/install_package virt-manager
 
+# missing package on ubuntu server installation
+tools/install_package make
+tools/install_package genisoimage
```

Comment:
Although I haven't changed a lot, I guess this is interesting for you guys @DukeHarris @espspinix @snrlx :
On ubuntu server this [tiny command](https://github.com/MasterprojectOpenStack1516/sourcecode/blob/ubuntu_server/installation/tools/create_vm_named#L70) exits with errors because no ```genisoimage``` command is found. It doesn't terminate the script unfortunately.